Gemfile

```diff
@@ -23,8 +23,8 @@ gem "rails", "~> 5.1.0"
 gem "resque"
 gem "resque-pool"
 
-gem "checkpoint", github: "mlibrary/checkpoint"
-gem "keycard", github: "mlibrary/keycard"
+gem "checkpoint", '~> 1.0.3', github: 'mlibrary/checkpoint', tag: 'v1.0.3'
+gem "keycard", "~> 0.3.0", github: 'mlibrary/keycard', branch: 'PFDR-106/api_key'
```

Comment:
I just merged and released 0.3.0, so this should be changed to the tag.

I guess we can just use the tags if we have to keep these as github sources... I seem to remember being able to use a plain version number _and_ `bundle config local.GEMNAME` somehow, but it eludes me today.